script.js

```diff
@@ -1 +1,132 @@
 // Please implement exercise logic here
+// keep data about the game in a 2-D array
+const board = [
```

Comment:
let should be used here instead of const because board is reassigned in line 107